lib/embed/media_tag.rb

```diff
@@ -149,10 +160,6 @@ def enabled_streaming_types
       Settings.streaming[:source_types]
     end
 
-    def file_index
```

Comment:
This is an interesting change.  I wasn't completely happy with its implementation, but it seemed to work.

Was there something not working with this under our new approach that requires us to pass this index around as an argument to 4 methods in the class?